<?xml version="1.0" encoding="UTF-8"?>

<launch>

  <node pkg="eagleye" name="velocity_scale_factor" type="velocity_scale_factor" />
  <node pkg="eagleye" name="yawrate_offset_stop" type="yawrate_offset_stop" />
  <node pkg="eagleye" name="yawrate_offset_1st" type="yawrate_offset" args="1st"/>
  <node pkg="eagleye" name="yawrate_offset_2nd" type="yawrate_offset" args="2nd"/>
  <node pkg="eagleye" name="heading_1st" type="heading" args="1st"/>
  <node pkg="eagleye" name="heading_2nd" type="heading" args="2nd"/>
  <node pkg="eagleye" name="heading_3rd" type="heading" args="3rd"/>
  <node pkg="eagleye" name="heading_interpolate_1st" type="heading_interpolate" args="1st"/>
  <node pkg="eagleye" name="heading_interpolate_2nd" type="heading_interpolate" args="2nd"/>
  <node pkg="eagleye" name="heading_interpolate_3rd" type="heading_interpolate" args="3rd"/>
  <node pkg="eagleye" name="distance" type="distance" />
  <node pkg="eagleye" name="trajectory" type="trajectory" />
  <node pkg="eagleye" name="position" type="position" />
  <node pkg="eagleye" name="position_interpolate" type="position_interpolate" />
  <node pkg="eagleye" name="debug" type="debug" />

</launch>
